plugins/Tools/CameraTool/CameraTool.py

```diff
@@ -40,6 +41,15 @@ def __init__(self):
 
         self._drag_distance = 0.05
 
+        Preferences.getInstance().addPreference("view/invert_zoom", False)
+        self._invert_zoom = Preferences.getInstance().getValue("view/invert_zoom")
+        Preferences.getInstance().preferenceChanged.connect(self._onPreferencesChanged)
+
+    def _onPreferencesChanged(self, name):
+        if name != "view/invert_zoom":
+            return
```

Comment:
I guess it would've made sense to just go:

    if name == "view/invert_zoom":
        self._invert_zoom = Preferences.getInstance().getValue("view/invert_zoom")

But this is okay too. Not blocking.